lrpar/examples/calc_wasm/LICENSE-APACHE

```diff
@@ -0,0 +1,10 @@
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+this file except in compliance with the License.  You may obtain a copy of the
+License at
+
```

Comment:
`wasm-pack` is pretty adamant that `LICENSE` files exist.

Reply:
If we really have to do this, we'll need to add the LICENSE-MIT too. But is there any way to shut wasm-pack up on this? Or, maybe, we just `symlink` back to the top-level files, to at least avoid bloating the repository?

Reply:
I'd thought about symlinks, but IIRC git doesn't do symlinks well on windows.
I forget what it does exactly (either it puts the destination path of the symlink in the contents, or just bloats the repository how we'd be trying to avoid anyways) but I don't think it had a helpful behavior.

I'll look and see if I can't somehow get wasm-pack to pipe down.

Reply:
There doesn't seem to be any workaround, there has been issues for 6 years, and PRs for at least 3.
I'll look and see though if there is anything I can do that is more similar to what we did with wasmtime,
setting a runner, so we can just avoid wasm-pack all together, as we aren't really using it for it's `pack into a tarball for distribution` behavior (where checking licenses makes sense).

I'm not certain this is going to be doable, e.g. the `trunkrs` project implements a lot of wasm-pack's packing behavior, but still doesn't seem to have it's own testing strategy afaict.

Reply:
Strangely and unexpectedly this appears to have been fixed by moving it into cttests in [c5a4efd](https://github.com/softdevteam/grmtools/pull/527/commits/c5a4efdebab5ee4148203551bdfb1baefec64aab).

There must be something different about the Cargo.toml that makes it not run the check or maybe being one directory closer to the `../Cargo.toml` instead of `../../Cargo.toml`, I would guess that is it.

Reply:
I did manage to get rid of wasm-pack fully in [da72a4d](https://github.com/softdevteam/grmtools/pull/527/commits/da72a4d59328363e05cfa750e14eeb4c70b8c285)